packages/kit-bg/src/vaults/impls/evm/sdkEvm/BaseApiProvider.ts

```
@@ -0,0 +1,612 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+
+import BigNumber from 'bignumber.js';
+import { md5 } from 'js-md5';
+import { isEmpty, isNaN, keyBy, omit, orderBy, uniqBy } from 'lodash';
```

Biome

[error] Do not shadow the global "isNaN" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)